src/scenic/syntax/translator.py

```diff
@@ -1824,6 +2073,39 @@ def visit_Call(self, node):
 		"""
 		func = node.func
 		newArgs = []
+
+		# wrap temporal requirement in lambda
```

Comment:
Please add an entry at the start of the list in the docstring explaining what this does, since it's a new transformation done by `visit_Call`.